docs/source/tutorials/datalab.ipynb

```diff
@@ -0,0 +1,1393 @@
+    "</div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install and import required dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`Datalab` has additional dependencies that are not included in the standard installation of cleanlab.\n",
+    "\n",
+    "You can use pip to install all packages required for this tutorial as follows:\n",
+    "\n",
+    "```\n",
+    "!pip install matplotlib \n",
+    "!pip install cleanlab[datalab]\n",
+    "\n",
+    "# Make sure to install the version corresponding to this tutorial\n",
+    "# E.g. if viewing master branch documentation:\n",
+    "#     !pip install git+https://github.com/cleanlab/cleanlab.git\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "dependencies = [\"cleanlab\", \"matplotlib\", \"datasets\"]  # TODO: make sure this list is updated\n",
+    "\n",
+    "if \"google.colab\" in str(get_ipython()):  # Check if it's running in Google Colab\n",
+    "    %pip install cleanlab  # for colab\n",
+    "    cmd = ' '.join([dep for dep in dependencies if dep != \"cleanlab\"])\n",
+    "    %pip install $cmd\n",
+    "else:\n",
+    "    missing_dependencies = []\n",
+    "    for dependency in dependencies:\n",
+    "        try:\n",
+    "            __import__(dependency)\n",
+    "        except ImportError:\n",
+    "            missing_dependencies.append(dependency)\n",
+    "\n",
+    "    if len(missing_dependencies) > 0:\n",
+    "        print(\"Missing required dependencies:\")\n",
+    "        print(*missing_dependencies, sep=\", \")\n",
+    "        print(\"\\nPlease install them before running the rest of this notebook.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: remove cell before merging to master\n",
+    "# !pip install git+https://github.com/cleanlab/cleanlab.git#egg=cleanlab[datalab]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.model_selection import cross_val_predict\n",
+    "\n",
+    "from cleanlab import Datalab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Create and load the data (can skip these details)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll load a toy classification dataset for this tutorial. The dataset has two numerical features and a label column with three classes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```

Comment:
plot could be nicer with two separate legends, one is titled: "Given Class Label" (contains the low, med, high info). The other legend is titled: "Type of issue" (contains the: Label Error, Outlier, Duplicates info).

What do you think? Currently, low/med/high is confusing if somebody just glances at the plot.

Reply:
Agree that it could be a little confusing at first glance - updated to include two legend with separate titles